proto/grp.proto

```diff
@@ -39,6 +41,16 @@ message Group {
   // Whether builds for this group will use custom executors provided by the
   // group.
   bool use_group_owned_executors = 7;
+
+  // Role represent's the authenticated user's role within this group, which
+  // determines which actions the user is authorized to perform.
+  Role role = 9;
+
+  enum Role {
+    UNKNOWN_ROLE = 0;
+    DEVELOPER_ROLE = 1;
+    ADMIN_ROLE = 2;
```

Comment:
comment at a high level what permissions these roles will entail or what the difference is?

Reply:
Done